### ISSUE TITLE
Enable multi-select + drag

### ISSUE THREAD

**Issue by PropGit** [parallaxinc/solo#396]
I think multi-select and drag a group of blocks was once possible (in the early days of BlocklyProp).  

Can we try enabling this again (if it's a feature that was turned off)?

**Comment by MatzElectronics** [comment]
No, it wasn't.  It's never been an option, and I don't believe google/blockly has any intention of supporting it.
